File: androidsupport/src/main/java/com/jarklee/androidsupport/lifecycle/LifeCycleHook.java
/*
 * ******************************************************************************
 *  Copyright Ⓒ 2017. TrinhQuan. All right reserved
 *  Author: TrinhQuan. Created on 2017/1/6
 *  Contact: trinhquan.171093@gmail.com
 * ******************************************************************************
 */

package com.jarklee.androidsupport.lifecycle;

import com.jarklee.androidsupport.callback.LifeCycleListener;

public interface LifeCycleHook {

    void detachListener(LifeCycleListener listener);
}

File: androidsupport-kotlin/src/main/java/com/jarklee/androidsupport/lifecycle/LifeCycleHook.kt
/*
 * ******************************************************************************
 *  Copyright Ⓒ 2016. TrinhQuan. All right reserved
 *  Author: TrinhQuan. Created on 2016/12/20
 *  Contact: trinhquan.171093@gmail.com
 * ******************************************************************************
 */

package com.jarklee.androidsupport.lifecycle

import com.jarklee.androidsupport.callback.LifeCycleListener

interface LifeCycleHook {

    fun removeLifeCycleHook(listener: LifeCycleListener)
}
